site/src/Librecores/ProjectRepoBundle/Resources/views/Project/settings.html.twig

```diff
@@ -67,6 +74,8 @@
       <h3 class="panel-title">Source Code Repository</h3>
     </div>
     <div class="panel-body">
+    <p><i>The link to your source code repository. This will be added as link on the project overview and
```

Comment:
No, that's the URL where you can do a checkout/clone. It usually it not linked, since the checkout/clone URL doesn't need to be web-accessible (that's a github special case).